include/circt/EmitVerilog.h

```diff
@@ -1,25 +0,0 @@
-//===- EmitVerilog.h - Verilog Emitter --------------------------*- C++ -*-===//
```

Comment:
nit but it would be nice to 'git mv' this file to retain history.  I think there is a post-facto way to do that with git.

Reply:
I don't know of a way to do it.  Since I modified the file quite a bit, it defeats git's rename detection heuristic. I don't think git stores rename information in a commit, but detects/recalculates it on the fly. The usual solution is to modify and move files in separate commits. I tried removing the license and forward declaration changes, but it still wasn't detected as a rename.

I could be totally wrong.  If there is a way to force git to detect a moved file I would love to know.

Reply:
Ok, don't worry about it, thanks